scripts/aws/create-kubernetes-cluster.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python
```

Comment:
I would always prefer `go` if possible.
@edwarnicke @fkautz any opinions here?

Reply:
I discussed it with Ed and changed scripts to Go lang. 
I keep python scripts in repository if somebody will need it.